types/mapbox-gl/index.d.ts

```diff
@@ -1796,6 +1810,10 @@ declare namespace mapboxgl {
         'text-allow-overlap'?: boolean;
         'text-ignore-placement'?: boolean;
         'text-optional'?: boolean;
+        'text-radial-offset'?: number | Expression;
+        'text-variable-anchor'?: Anchor[];
+        'text-writing-mode'?: ['horizontal', 'vertical'];
```

Comment:
I'm not sure about this one should not it be ('horizontal' | 'vertical')[] instead?

Reply:
I really missed that one. I’ll fix it 👍